cpg_workflows/stages/dragen_ica/run_align_genotype_with_dragen.py

```diff
@@ -45,31 +44,28 @@ def submit_dragen_run(
         analysisInput=NextflowAnalysisInput(
             inputs=[
                 AnalysisDataInput(parameterCode='crams', dataIds=[cram_id]),
-                AnalysisDataInput(parameterCode='crais', dataIds=[cram_index_id]),
                 AnalysisDataInput(parameterCode='ref_tar', dataIds=[dragen_ht_id]),
                 AnalysisDataInput(parameterCode='cram_reference', dataIds=[cram_reference_id]),
                 AnalysisDataInput(parameterCode='qc_cross_cont_vcf', dataIds=[qc_cross_cont_vcf_id]),
-                AnalysisDataInput(
-                    parameterCode='qc_coverage_region_beds',
-                    dataIds=[qc_cov_region_1_id, qc_cov_region_2_id],
-                ),
+                AnalysisDataInput(parameterCode='qc_coverage_region_1', dataIds=[qc_cov_region_1_id]),
+                AnalysisDataInput(parameterCode='qc_coverage_region_2', dataIds=[qc_cov_region_2_id]),
             ],
             parameters=[
-                AnalysisParameterInput(code='enable_map_align', value='True'),
-                AnalysisParameterInput(code='enable_map_align_output', value='True'),
+                AnalysisParameterInput(code='enable_map_align', value='true'),
+                AnalysisParameterInput(code='enable_map_align_output', value='true'),
                 AnalysisParameterInput(code='output_format', value='CRAM'),
-                AnalysisParameterInput(code='enable_duplicate_marking', value='True'),
-                AnalysisParameterInput(code='enable_variant_caller', value='True'),
-                AnalysisParameterInput(code='vc_emit_reference_confidence', value='GVCF'),
-                AnalysisParameterInput(code='vc_enable_vcf_output', value='False'),
-                AnalysisParameterInput(code='qc_coverage_report', value='cov_report,cov_report'),
-                AnalysisParameterInput(code='qc_coverage_filters', value='mapq<1,bq<0,mapq<1,bq<0'),
-                AnalysisParameterInput(code='enable_cnv', value='True'),
-                AnalysisParameterInput(code='cnv_enable_self_normalization', value='True'),
-                AnalysisParameterInput(code='enable_sv', value='True'),
+                AnalysisParameterInput(code='enable_duplicate_marking', value='true'),
+                AnalysisParameterInput(code='enable_variant_caller', value='true'),
+                AnalysisParameterInput(code='vc_emit_ref_confidence', value='GVCF'),
+                AnalysisParameterInput(code='vc_enable_vcf_output', value='false'),
+                AnalysisParameterInput(code='enable_cnv', value='true'),
+                AnalysisParameterInput(code='cnv_segmentation_mode', value='SLM'),
```

Comment:
Confirming for the record that this new parameter has a default value of 'SLM' for [germline WGS analysis](https://support.illumina.com/content/dam/illumina-support/documents/documentation/software_documentation/dragen-bio-it/Illumina-DRAGEN-Bio-IT-Platform-User-Guide-1000000141465-00.pdf), and so this should be consistent with UK Biobank / All of Us (even though it isn't explicitly recorded [here](https://developer.illumina.com/dragen/dragen-popgen).